src/PerimeterxCookie.php

```diff
@@ -2,191 +2,6 @@
 
 namespace Perimeterx;
 
-abstract class PerimeterxCookie {
+abstract class PerimeterxCookie extends PerimeterxPayload {
```

Comment:
I don't see the value of adding another layer of abstraction with `PerimeterxCookie` and `PerimeterxToken`. If they don't add any distinct functionality, we can just get rid of them.